kubernetes/gke-utility/argocd/clusters.yaml

```diff
@@ -23,3 +24,30 @@ stringData:
       }
     }
 ---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: gke-prow-build
```

Comment:
What's this used for?

Reply:
@upodroid I was wondering what that kubeconfig is used, I think I might asked the same in the previous PR but forgot 😅